core/src/test/resources/org/bitcoinj/script/script_invalid.json

```diff
@@ -151,8 +151,8 @@
 ["2 2 LSHIFT", "8 EQUAL", "P2SH,STRICTENC", "disabled"],
 ["2 1 RSHIFT", "1 EQUAL", "P2SH,STRICTENC", "disabled"],
 
-["1","NOP1 NOP2 NOP3 NOP4 NOP5 NOP6 NOP7 NOP8 NOP9 NOP10 2 EQUAL", "P2SH,STRICTENC"],
```

Comment:
Sure, you're right. I'll merge the two files and look at the tests there.
I've just checked the original tests about CHECKSEQUENCEVERIFY. I wonder how they deal with tx version and tx input sequence. Any hint?